### ISSUE TITLE
“TypeError: r is undefined” when geolocalizing contact

### ISSUE THREAD

**Issue by clochix** [RobyRemzy/cozy-map#25]
Here's the error I get into the console when I try to see one of my contacts on the map:
```
process for:  158 rue de Verdun  92800 Puteaux  FRANCE  loading... map.min.js:28037:7
TypeError: r is undefined
 map.min.js:28045:35
error with conctat address!
```

It would be nice to catch the error and provide some user feedback ;-)

**Comment by jojorb** [comment]
yes, it happened when the address is not well known by Nominatim with openstreetmap, like mostly the house number, or a miss spell.  I was thinking about that error a lot but the solutions would be 1- to edit osm node and enter the correct location informations or 2- use a second reverse geocoding (probably provided with a token) to get a second fallback for the best.

**Comment by clochix** [comment]
I think we have two different issues here:
- a JS exception that is not trapped;
- no feedback to user when the library is unable to find the location.

Not being able to find a location is ok, but we should display something in the UI to explain to the user what's happening.
